vmdb/app/controllers/miq_request_controller.rb

```diff
@@ -91,10 +97,19 @@ def request_edit
       @req_id = params[:id]
       @prov_type = prov.resource_type
     else
-      session[:checked_items] = prov.options[:src_ids]
-      @refresh_partial = "reconfigure"
-      @_params[:controller] = "vm"
-      reconfigurevms
+      if prov.kind_of?(MiqProvisionConfiguredSystemRequest)
+        @org_controller = "configured_system"
```

Comment:
Ok, made that change.
Also refactored ```request_edit``` and ```request_copy``` (extracted common code in private method)